low-level/framework/base/sspl_constants.py

```diff
@@ -59,6 +59,7 @@
 # required only for init
 component = 'sspl/config'
 file_store_config_path = '/etc/sspl.conf'
+roles=['vm', 'cortx']
```

Comment:
so should I include <ssu | gw | cmu> here??

Reply:
yeah